src/lib/y2storage/autoinst_profile/drive_section.rb

```diff
@@ -49,7 +49,8 @@ def self.attributes
           { name: :pesize },
           { name: :type },
           { name: :use },
-          { name: :skip_list }
+          { name: :skip_list },
+          { name: :raid_options }
```

Comment:
I'm just starting the review but, why has been this added to `<drive>`? I don't see the reason at first sight

Reply:
Because we have introduced partitioned RAIDs. In the past, each partition section defined a different RAID, so the RAID options were defined at that level.

However, now that each partition can define a partition itself, I think we should define these options at `drive` level.